mirgecom/initializers.py

```diff
@@ -427,12 +592,15 @@ def __call__(self, t, x_vec, eos=IdealSingleGas()):
         eos: :class:`mirgecom.eos.GasEOS`
             Equation of state class to be used in construction of soln (unused)
         """
-        gamma = eos.gamma()
-        mass = x_vec[0].copy()
-        mom = self._velocity * make_obj_array([mass])
-        energy = (self._p / (gamma - 1.0)) + np.dot(mom, mom) / (2.0 * mass)
-
-        return flat_obj_array(mass, energy, mom)
+        return _make_uniform_flow(x_vec=x_vec, mass=self._rho,
+                                  pressure=self._p, energy=self._e,
+                                  eos=eos)
+        # gamma = eos.gamma()
+        # mass = x_vec[0].copy()
+        # mom = self._velocity * make_obj_array([mass])
+        # energy = (self._p / (gamma - 1.0)) + np.dot(mom, mom) / (2.0 * mass)
+        #
+        # return flat_obj_array(mass, energy, mom)97
```

Comment:
* How come this is retained?
* What's with the trailing 97?

Reply:
I do not know.